lms/static/sass/course/courseware/_sidebar.scss

```diff
@@ -108,40 +100,46 @@
       }
     }
 
-    ul.ui-accordion-content {
-      background: transparent;
+    div.chapter-content-container{
+      display: none;
+      padding: 0 14px ($baseline/2) 14px;
+      border-bottom: 1px solid $shadow-l1;
+      background: $white;
+    }
+
+    div.ui-accordion-content {
+      margin: 0;
+      padding: 0 ($baseline*.75);
       border: none;
       border-radius: 0;
-      margin: 0;
-      padding: 9px 0 9px 9px;
-      overflow: auto;
-      width: 100%;
+      background: transparent;
 
       li {
+        margin-bottom: ($baseline/5);
         border-bottom: 0;
         border-radius: 0;
-        margin-bottom: ($baseline/5);
 
         a {
-          background: transparent;
-          border-radius: 4px;
-          display: block;
-          @include padding( ($baseline/4), ($baseline*1.5), ($baseline/4), ($baseline/2));
+          @extend %t-strong;
+          @include padding(($baseline/4) ($baseline/2));
           position: relative;
+          display: block;
+          border-radius: ($baseline/5);
+          background: transparent;
           text-decoration: none;
 
           p {
-            font-weight: bold;
-            font-family: $sans-serif;
+            @extend %t-action3;
+            @extend %t-strong;
             margin-bottom: 0;
-            line-height: 1.3;
+            font-family: $sans-serif;
```

Comment:
NIT: should our t-action extends also include a base font-family reset like copy / title extends? If so we could drop this line but probably not necessary here. Just a comment in passing.